spec/src/main/asciidoc/5_interceptor_ordering.adoc

```diff
@@ -36,7 +36,7 @@ overridden.
 
 An extension specification may define
 alternative mechanisms (e.g., a deployment descriptor such as the CDI
-beans.xml [<<bib3>>] or the EJB _ejb-jar.xml_
+beans.xml [<<bib3>>] or the Enterprise Jakarta Beans _-jar.xml_
```

Comment:
The file is still called _ejb-jar.xml_, which is one of the exceptions where we could continue using the term.

So it should then become Enterprise Jakarta Beans _ejb-jar.xml_

Reply:
Done